includes/gateway/class-omise-payment-creditcard.php

```diff
@@ -297,20 +297,12 @@ public function process_payment( $order_id ) {
 
 				$order->add_order_note( sprintf( __( 'Omise: Charge (ID: %s) has been created', 'omise' ), $charge['id'] ) );
 
-				$this->attach_charge_id_to_order( $charge['id'] );
+				$this->set_order_transaction_id( $charge['id'] );
```

Comment:
why this name was changed?

Reply:
@jacstn  Originally it was named `attach_charge_id_to_order` as its intention, to attach Omise charge id to WC's order item so we can retrieve it back later for those offsite payments (3-D Secure included).

Then, it comes to the second version, where I integrated `Omise charge id` with `transaction id` feature provided by WC as you can see from the following code:
```php
if ( version_compare( WC()->version, '3.0.0', '>=' ) ) {
	$order->set_transaction_id( $charge['id'] );
	$order->save();
} else {
	update_post_meta( $order->id, '_transaction_id', $charge['id'] );
}
```
However, these 2 code are redundant to each other. So, I removed these 2 code, `attach_charge_id_to_order`, and the above code. Then having `set_order_transaction_id` method, that simply to the job as its name (set order transaction id).